jax/interpreters/masking.py

```diff
@@ -1,74 +1,57 @@
-# Copyright 2019 Google LLC
```

Comment:
Accidentally deleted the copyright notice?

Reply:
I wanted to run Google-internal tests before merging, to make sure we don't break anything, but it turns out I can't sync the code down to Google until the copyright message is restored!

Reply:
Now fixed, was lost while moving around code between files and back.